jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -43,14 +43,12 @@ describe('Charmbrowser', function() {
         }
       }};
     var changeState = sinon.stub();
-    var addService = sinon.stub();
     var charmstoreSearch = sinon.stub();
     var output = jsTestUtils.shallowRender(
       <juju.components.Charmbrowser
         appState={appState}
         changeState={changeState}
-        charmstoreSearch={charmstoreSearch}
-        addService={addService} />);
```

Comment:
I added `addService` last week, you're fine to modify it as you see fit, if you don't end up using it, please remove it.